optaweb-vehicle-routing-benchmark/src/main/java/org/optaweb/vehiclerouting/benchmark/DataSetGenerator.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.vehiclerouting.benchmark;
+
+import static java.util.stream.Collectors.joining;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.IntStream;
+
+import org.optaweb.vehiclerouting.domain.Coordinates;
+import org.optaweb.vehiclerouting.domain.Location;
+import org.optaweb.vehiclerouting.domain.LocationData;
+import org.optaweb.vehiclerouting.domain.RoutingProblem;
+import org.optaweb.vehiclerouting.service.distance.DistanceCalculationException;
+import org.optaweb.vehiclerouting.service.distance.DistanceCalculator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class DataSetGenerator {
+
+    private static final Logger logger = LoggerFactory.getLogger(DataSetGenerator.class);
+    private static final int MAX_TRIES = 10;
+    private final DistanceCalculator distanceCalculator;
+    private final RoutingProblem problem;
+    private final int maxDescriptionLength;
+
+    DataSetGenerator(DistanceCalculator distanceCalculator, RoutingProblem problem) {
+        this.distanceCalculator = distanceCalculator;
+        this.problem = problem;
+        maxDescriptionLength = problem.visits().stream().mapToInt(v -> v.description().length()).max().orElse(0);
+    }
+
+    static LocationData nextDatSetItem(RoutingProblem problem, int i) {
+        return problem.visits().get(i % problem.visits().size());
+    }
+
+    static Coordinates randomize(Coordinates coords) {
+        return Coordinates.valueOf(
+                coords.latitude().doubleValue() + Math.random() * 0.08 - 0.04,
```

Comment:
Could you please explain the numbers here?

Reply:
That's ± 0.04° (± 4,4 km).

Reply:
Thanks, maybe a short comment would help.